scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -58,6 +58,30 @@ package object dialects {
     .withAllowNumericLiteralUnderscoreSeparators(true)
     .withAllowTryWithAnyExpr(true)
 
+  /**
+   *  Dialect for Scala 2.13.6 with `-Xsource:3` option
+   */
+  implicit val Scala213Source3 = Scala213
+    .withAllowQuestionMarkPlaceholder(true)
```

Comment:
I haven't added `case` in for expression since we don't actually have a separate option for that, I can add it if needed later, but it's rather a minor problem - it will parse it always currently.